.github/ISSUE_TEMPLATE/generic-bug.md

```diff
@@ -24,7 +24,7 @@ The issue happens in the following browser(s): _____
 
 **Questions**:
 
-- [ ] I have verified the bug exists in the latest version of `@synergy-design-system/core` (e.g. via `npm install @synergy-design-system/core@latest`).
+- [ ] I have verified the bug exists in the latest version of `@synergy-design-system/core` (e.g. via `npm install @synergy-design-system/core@latest`). <!-- core??? -->
```

Comment:
Good catch ;). Seems like a leftover from the original version. Should be `@synergy-design-system/components`.